zebra/explorer.md

```diff
@@ -13,23 +13,23 @@ https://zcashblockexplorer.com/
+| getrawtransaction           | YES, partially.    | The `blockhash` argument is not used in the explorer which is good but at least the following extra fields are needed in the response:<br>`txid`<br>`time` (from block header)<br>`size`<br>`vout`<br>tx type: many transparent and shielded fields<br>We'll need to add `tx.size` to the existing database and non-finalized state indexes.               | [Zcash](https://zcash.github.io/rpc/getrawtransaction.html) - [Zebra](https://doc.zebra.zfnd.org/zebra_rpc/methods/trait.Rpc.html#tymethod.get_raw_transaction)              | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/transactions/transaction_cache.ex#L30) - [2](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/search_controller.ex#L15) - [3](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/transaction_controller.ex#L5) [called by](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/live/recent_transactions_live.ex#L19) - [4](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/transaction_controller.ex#L12)                                                                                                     |
+| getaddressbalance           | YES, partially.    | Zebra needs to implement the `received` field.<br>This needs to be stored for each address in the database, as a `u128`.                                                                                  | [Zcash](https://zcash.github.io/rpc/getaddressbalance.html) - [Zebra](https://doc.zebra.zfnd.org/zebra_rpc/methods/trait.Rpc.html#tymethod.get_address_balance)              | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/address_controller.ex#L28) - [2](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/address_controller.ex#L72) [calls](https://github.com/nighthawk-apps/zcashex/blob/main/lib/zcashex.ex#L128) [called by](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/templates/address/address.html.eex#L45)                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
The arguments are always an array, `zcashd` doesn't accept a single string. (The docs are wrong!) Linked to the code that changes a single argument to an array.

We also need an extra field here.